dev/com.ibm.ws.wsoc/resources/com/ibm/ws/wsoc/internal/resources/WebSockets.nlsprops

```diff
@@ -228,9 +223,9 @@ missingslashornull.uri=CWWKH0052E: An exception occurred during deployment/start
 missingslashornull.uri.explanation=ServerEndpoint is invalid because Endpoint URI is either null or does not start with / in endpoint class. WebSocket application is not available for requests.
 missingslashornull.uri.useraction=Define non null Endpoint URI beginning with /. After you resolve the problem, deploy the application again.
 
-wsoc.feature.not.loaded.correctly=CWWKH0053E: The websocket feature did not load with an acceptable version.
-wsoc.feature.not.loaded.correctly.explanation=The websocket feature is loaded with a version that corresponds to the Java [Jakarta] EE specification of that websocket feature. The feature was loaded with no version defined.
-wsoc.feature.not.loaded.correctly.useraction=Check the server.xml file for the existence of multiple websocket features. Only one should be specified.
+wsoc.feature.not.loaded.correctly=CWWKH0053E: The WebSocket container did not load with an acceptable version.
```

Comment:
Thoughts on keeping it WebSocket feature? This seems consistent with Pages: https://github.com/OpenLiberty/open-liberty/pull/22709/files

Servlet, we reference on a regular basis the "servlet container", is the same true for WebSocket?

Reply:
Should this also apply to servlet? 
It is currently: `The servlet container did not load with an acceptable version.` 

However, I'll the explanation to mention the container?